debian/vyos-1x.postinst

```diff
@@ -180,6 +180,9 @@ systemctl enable vyos-config-cloud-init.service
 # Generate API GraphQL schema
 /usr/libexec/vyos/services/api/graphql/generate/generate_schema.py
 
+# Update XML cache
+python3 /usr/lib/python3/dist-packages/vyos/xml_ref/update_cache.py
```

Comment:
As there is a #shebang inside `update_cache.py` why call it with the python3 interpreter?

Reply:
Simply a permissions issue: otherwise one needs to set +x, as python3 setup install unsets executable permissions, and in *.postinst it is needed (though not in Makefile); good point the shebang is not then needed in update_cache.py.

Reply:
^unsets^ignores ... note that `generate_schema.py` is not installed via setuptools, so the executable permissions are retained. I'd prefer to leave it for now, as there may be further reorganization depending on the needs of the vpp package.

Reply:
It was just a note - please continue.